example/index.android.js

```diff
@@ -0,0 +1,69 @@
+/**
+* Sample React Native App
+* https://github.com/facebook/react-native
+* @flow
+*/
+
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View,
+  TouchableHighlight,
+  NativeModules,
+  NativeEventEmitter
+} from 'react-native';
+
+import RazorpayCheckout from 'react-native-razorpay';
+
+class example extends Component {
+
+  render() {
+    return (
+      <View style={styles.container}>
+      <TouchableHighlight onPress={() => {
+        var options = {
+          description: 'Credits towards consultation',
+          image: 'https://i.imgur.com/3g7nmJC.png',
+          currency: 'INR',
+          key: 'rzp_test_1DP5mmOlF5G5ag',
+          amount: '5000',
+          name: 'foo',
+          prefill: {
+            email: 'akshay@razorpay.com',
+            contact: '8955806560',
+            name: 'Akshay Bhalotia'
+          },
+          theme: {color: '#F37254'}
+        }
+        RazorpayCheckout.open(options).then((data) => {
+          // handle success
+          alert(`Success: ${data.payment_id}`);
+        }).catch((error) => {
+          // handle failure
+          alert(`Error: ${error.code} | ${error.description}`);
+        });
+      }}>
+      <Text style = {styles.text}>Pay</Text>
+      </TouchableHighlight>
+      </View>
+    );
+  }
+
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#F5FCFF',
+  },
+  text: {
+    fontSize: 20,
+    fontWeight: 'bold',
+  }
+});
+
+AppRegistry.registerComponent('example', () => example);
```

Comment:
Each platform requires its own entry file. We can point both entrypoint files (`index.android.js` and `index.ios.js`) to the common code in `index.js` to avoid code duplication